output/github.go

```diff
@@ -0,0 +1,94 @@
+package output
+
+import (
+	"fmt"
+	"io"
+)
+
+// GitHub represents an Outputter that outputs
+// results in GitHub workflow format.
```

Comment:
a link to the spec would be useful here

Reply:
Added a link to the GitHub documentation on workflow commands.